sklearn/linear_model/tests/test_sparse_coordinate_descent.py

```diff
@@ -267,3 +267,36 @@ def test_same_output_sparse_dense_lasso_and_enet_cv():
         assert_almost_equal(clfs.intercept_, clfd.intercept_, 7)
         assert_array_almost_equal(clfs.mse_path_, clfd.mse_path_)
         assert_array_almost_equal(clfs.alphas_, clfd.alphas_)
+
+
+def test_same_multiple_output_sparse_dense():
+    X, y = make_sparse_data(n_samples=40, n_features=10)
```

Comment:
am i missing something, or do you not use the data generated with `make_sparse_data`?